pkg/sharding_engine/thick_client.go

```diff
@@ -62,16 +62,21 @@ func NewThickClient(clientId string, rpc pb.SeClientApiClient) (ThickClient, *st
 
 //todo: this should be a push instead of poll
 func (t *tc) syncSe() error {
-	rs, err := t.rpc.AllParts(context.TODO(), &pb.AllPartsReq{ClientId: t.clientId})
+	rs, err := t.rpc.GetPartitions(context.TODO(), &pb.PartitionsGetRequest{})
 	if err != nil {
 		return err
 	}
 
-	newTime := rs.GetLastModified()
+	newTime := int64(0)
```

Comment:
`newTime = 0` will always update

Reply:
`	if newTime <= t.cache.LastModified() {
		return nil
	}
`

this check will prevent it. but i missed to update the timestamp in cache, i fixed it now.